### PR TITLE
doc(zh-cn): translate deploy

### PR DESCRIPTION
**Pull request by kecrily**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

translate deploy

### CODE REVIEW

File: src/pages/zh-cn/guides/deploy.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+title: 部署你的 Astro 站点
+description: 如何在互联网上部署你的 Astro 站点
+layout: ~/layouts/MainLayout.astro
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+---
+**准备好构建并部署你的 Astro 站点了吗？**跟着任一部署服务指南走，或向下滚动查看关于部署 Astro 网站的普适性指引。
```
Comment (delucis) [inline src/pages/zh-cn/guides/deploy.md:7]: Looks like a missing space here is messing up Markdown parsing. Would it be valid to add a space like this in Chinese?

```suggestion
**准备好构建并部署你的 Astro 站点了吗？** 跟着任一部署服务指南走，或向下滚动查看关于部署 Astro 网站的普适性指引。
```

File: src/pages/zh-cn/guides/deploy.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+这些服务商会自动检测到你所推送 Astro 项目的源码库，构建站点并将其部署到自定义 URL 或你的个人域名上。通常情况下，在这些平台上设置部署将遵循类似以下的步骤。
+
+1. 将你的仓库添加到在线 Git 提供商（如 GitHub、GitLab、Bitbucket）。
+
+1. 选择支持**持续集成**的服务商（例如 [Netlify](/zh-cn/deploy/netlify/) 或 [Vercel](/zh-cn/deploy/vercel/)），将你的 Git 仓库作为一个新站点/项目导入。
+
+    许多常见服务商会识别你的项目为 Astro 网站，并应选择适当的配置设置来构建和部署你的网站，如下所示。（如果不是，那么这些设置也可以进行修改）
+
+    ::注意[部署设置]
```
Comment (delucis) [inline src/pages/zh-cn/guides/deploy.md:27]: Same thing as the other PR, this should use the aside format:

```suggestion
    :::note[部署设置]
```

File: src/pages/zh-cn/guides/deploy.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+默认情况下，构建输出将会被放在 `dist/`. 你可以修改 [`outDir` 配置项](/zh-cn/reference/configuration-reference/#outdir)改变存储位置。
+
+## 添加 SSR 适配器
+
+:::note
+在使用 [SSR （服务端渲染）](/zh-cn/guides/server-side-rendering/) 部署你的 Astro 站点前，请确保：
+
+    - 在项目依赖中安装[合适的适配器](/zh-cn/guides/server-side-rendering/#enabling-ssr-in-your-project)
+    - 在你的 `astro.config.mjs 文件导入和默认导出中[加上了适配器](/zh-cn/reference/configuration-reference/#integrations)
```
Comment (delucis) [inline src/pages/zh-cn/guides/deploy.md:81]: Think we’re missing a backtick here:

```suggestion
    - 在你的 `astro.config.mjs` 文件导入和默认导出中[加上了适配器](/zh-cn/reference/configuration-reference/#integrations)
```

**Review by delucis — COMMENTED** [review]
You sure have been working hard! 😅 

Spotted a couple of things on this one.

**Review by delucis — APPROVED** [review]
Another page done! 🥳

**Comment by delucis** [comment]
One note @kecrily — we haven’t actually marked this and the other deploy guides as `i18nReady` yet, so I’d recommend not translating any of those yet. You can check #438 for all the pages that still need translating. I think a PR updating the `zh-cn` `ui.ts` and `nav.ts` files would also be helpful.

**Comment by kecrily** [comment]
I just now understand what `i18nReady` means.

I've been comparing missing files between file trees for translation.

**Comment by delucis** [comment]
Haha, oh no! Yes we have that beautiful issue that keeps track of things 💜

**Pull request merged**